### PR TITLE
Other Org : Add Validation for singleton Orgs. 

### PR DESCRIPTION
**Pull request by manukundloo-hmcts**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RDCC-7061

### Change description ###
Other Org : Add Validation for singleton Orgs. 

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ x] Does this PR introduce a breaking change

### CODE REVIEW

**Comment by manukundloo-hmcts** [comment]
Closing the Pr since it is failing will reopen again

File: src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java [hunk 1]
```diff
@@ -351,11 +362,31 @@ protected ResponseEntity<Object> updateOrganisationById(OrganisationCreationRequ
...
+
+            if (organisationCreationRequest instanceof OrganisationOtherOrgsCreationRequest orgCreationRequestV2) {
+                String orgTypeInRequest = orgCreationRequestV2.getOrgType();
+                Optional<SingletonOrgType> isOrgTypePresentInSingleTonOrgTable = singletonOrgTypeRepository
+                                                        .findByOrgType(orgTypeInRequest);
+
+                if (isOrgTypePresentInSingleTonOrgTable.isPresent()) {
+                    var organisationList = organisationRepository
+                            .findByOrgTypeAndStatus(orgTypeInRequest, ACTIVE);
+                    if (organisationList.size() == 1 && !organisationList.get(0)
```
[the remark this answers is not in the document]
Comment (manukundloo-hmcts) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java:383]: We have done the changes in service layer. Resolving the comment after the discussion.

**Comment by manukundloo-hmcts** [comment]
closing the PR since PR build is failing will reopen again

**Comment by manukundloo-hmcts** [comment]
Closing PR since failign for some issue

**Comment by manukundloo-hmcts** [comment]
Closing the PR since it is failing for Functional test cases.

**Review by vasthav-patta — APPROVED** [review]

**Comment by manukundloo-hmcts** [comment]
Closing this PR because FT are failing. Raised new Pr an 1461 which is passed now.

**Pull request closed without merging**